src/main/java/ch/wisv/areafiftylan/controller/OrderRestController.java

```diff
@@ -128,14 +147,21 @@ public Order getOrderById(@PathVariable Long orderId) {
     @PreAuthorize("@currentUserServiceImpl.canAccessOrder(principal, #orderId)")
     @RequestMapping(value = "/orders/{orderId}/checkout", method = RequestMethod.GET)
     public ResponseEntity<?> payOrder(@PathVariable Long orderId) throws URISyntaxException {
-        //TODO: Implement Paymentprovider calls here.
         String paymentUrl = orderService.requestPayment(orderId);
         HttpHeaders headers = new HttpHeaders();
         headers.setLocation(new URI(paymentUrl));
 
         return createResponseEntity(HttpStatus.OK, headers, "Please go to " + paymentUrl + " to finish your payment");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
+    @RequestMapping(value = "/orders/{orderId}/checkout", method = RequestMethod.POST, params = "admin")
```

Comment:
Why are we using this as url, the admin parameter is a bit out of place. Why not change the url to it's purpose, `/orders/{orderId}/approve`? A javadoc comment would also be very helpful in this case.